mods/ra/weapons/missiles.yaml

```diff
@@ -246,7 +246,6 @@ TorpTube:
 			Wood: 75
 			Light: 75
 			Heavy: 100
-			Concrete: 500
```

Comment:
This looks like an unrelated change?

Reply:
No. Bridges has Concrete armor and if i don't remove that subs destroy bridges too fast due to the fact that they now have proper HitShapes. This reduces it to be more similar to what it was vs what it is now.

Reply:
Why don't they show their hitshape when I enable the debug option?

Reply:
Because they don't have CombatDebugOverlay trait. Should i add it with this PR, i used it for testing a lot.

Reply:
Sounds like a good idea. :+1: